.golangci.yml

```diff
@@ -0,0 +1,77 @@
+# v1.43.0
```

Comment:
This is currently getting updated by @olegbespalov so maybe pull the new one

Reply:
@javaducky there you could find a PR https://github.com/grafana/k6/pull/2609/files

Reply:
.... and https://github.com/grafana/k6/pull/2611 :see_no_evil:

Reply:
Thanks guys!